bluesea_node:
  ros__parameters:
  #ROS2#
    frame_id: "map"
    scan_topic: "scan"
    cloud_topic: "cloud"
  #DATA#
    dev_id: -1
    raw_bytes: 3
    output_360: true
    output_scan: true
    output_cloud2: false
    with_angle_filter: false
    min_angle: -1.0
    max_angle: 2.0
    # mask1: "0,3.14"
    # mask2: "-3,-2"
    hard_resample: false
    soft_resample: false
    with_checksum: true
    reversed: false
    min_dist: 0.01
    max_dist: 40.0
    error_circle: 3
    error_scale: 0.9
    with_confidence: 1
    unit_is_mm: 0
  #CONNECT#
    type: "udp"
    lidar_ip: "192.168.158.98"
    lidar_port: 6543
    local_port: 6668
    group_ip: "224.1.2.3"
  #GET#
    uuid:  -1
  #SET#
    resample_res: -1.0
    with_confidence: -1
    with_smooth: -1
    with_deshadow: -1
    alarm_msg: -1
    rpm: -1
    direction:  -1
    ats: -1
